config/config.go

```diff
@@ -218,12 +244,18 @@ func WithRequestsType(requestsType string) Option {
 	}
 }
 
+// WithMinTLSVersion applies minimum TLS version that is acceptable.
+//
+// Deprecated: use explicit WithTLSConfig option instead
 func WithMinTLSVersion(minVersion uint16) Option {
 	return func(c *Config) {
 		c.tlsConfig.MinVersion = minVersion
```

Comment:
Here and in similar places - may be check c.tlsConfig != nil or force WithSecure(true) for prevent nil panic on config apply?

Reply:
You right, ack